src/less/chip/chip.less

```diff
@@ -0,0 +1,60 @@
+@import "../variables/variables.less";
+
+.chip {
+    align-items: center;
+    background-color: var(--color-background-tertiary);
+    border-radius: @spacing-400;
+    box-sizing: border-box;
```

Comment:
Hmm, is this needed? I know it can be used for height, but then padding top/bottom are not needed. 
Or we can set the padding top/bottom to like `4px` and remove the height and such. 
This also allows us to remove that `margin-top` on the text part because then the text will be completely centered if there is no height set.

Reply:
Yea, there are other ways of doing it. I like this approach as it ensures the height. without having to do the internal calculations and remains constant even if the font size inside changes.

Reply:
The text was completely centered. I simply nudged it a bit to get an optical center with the icon button.

Reply:
Gotcha. Yea, that scales better. Will make the changes.